gammapy/modeling/models/tests/test_cube.py

```diff
@@ -624,3 +626,20 @@ def test_sky_model_create():
     assert isinstance(m.spatial_model, PointSpatialModel)
     assert isinstance(m.spectral_model, PowerLawSpectralModel)
     assert m.name == "my-source"
+
+
+def test_integrate_geom():
+    model = GaussianSpatialModel(lon="0d", lat="0d", sigma=0.1*u.deg, frame='icrs')
+    spectral_model = PowerLawSpectralModel(amplitude="1e-11 cm-2 s-1 TeV-1")
+    sky_model = SkyModel(spectral_model=spectral_model, spatial_model=model)
+
+    center = SkyCoord("0d", "0d", frame='icrs')
+    radius = 0.3 * u.deg
+    square = CircleSkyRegion(center, radius)
+
+    axis = MapAxis.from_energy_bounds("1 TeV", "10 TeV", nbin=10, name='energy_true')
+    geom = RegionGeom(region=square, axes=[axis])
+
+    integral = sky_model.integrate_geom(geom).data
+    print(integral.sum())
```

Comment:
Please remove the `print` statement here.

Reply:
Oh sorry, my fault!